lib/common/tests/iso8601/pcmk__time_format_hr_test.c

```diff
@@ -73,6 +73,11 @@ no_specifiers(void **state)
                      "this has a literal %01 in it", NULL);
     assert_hr_format("%2 this starts and ends with %",
                      "%2 this starts and ends with %", NULL);
+
+    /* strftime() treats % with a number (and no specifier) as a literal string
+     * to be formatted with a field width (undocumented and probably a bug ...)
+     */
+    assert_hr_format("this ends with %10", "this ends with        %10", NULL);
```

Comment:
...what

agreed probably a bug

Reply:
The documented syntax doesn't allow for it, so technically it's "undefined behavior". A literal percent is supposed to be doubled. The buffer overflow in our code was for this syntax though, so I wanted to keep a test for it.